ballerina/Ballerina.toml

```diff
@@ -20,4 +20,7 @@ version = "1.6.1"
 path = "../native/build/libs/protobuf-native-1.6.1.jar"
 
 [[platform.java17.dependency]]
+groupId = "com.google.protobuf"
+artifactId = "protobuf-java"
+version = "3.25.5"
 path = "./lib/protobuf-java-3.25.5.jar"
```

Comment:
We should modify this file:
https://github.com/ballerina-platform/module-ballerina-protobuf/blob/21baa2055a66c8b96e55fb9c7d097b4c0efeb881/build-config/resources/Ballerina.toml#L23

And run the gradle build, then the `ballerina/Ballerina.toml` should get updated automatically in a commit

Reply:
Ah yes. Will fix that

Reply:
Fixed now